src/features/components/molecules/qr-scan-box/styles.module.scss

```diff
@@ -0,0 +1,4 @@
+.qr_box {
+    border: 0.5px solid var(--color-white);
+    border-radius: 24px;
```

Comment:
Is this size provided by design team? Could you please check in figma 🙏

Reply:
Yes, it is same as design.